apps/docs/src/lib/components/TopBar/TopBar.svelte

```diff
@@ -60,34 +62,61 @@
 		localStorage.setItem('recentSearches', JSON.stringify(existingSearches));
 	}
 
+	function changePreviousSection(newVal: string) {
+		previousSection = newVal;
+		return newVal;
+	}
+
+  function validateSearchLink(searchLinkSection: string) {
+    return searchLinkSection === previousSection
+  }
+
 	// @ts-nocheck
```

Comment:
can be removed?

Reply:
I tried but for some inexplicable reason when I do it, it results in this bug:

https://user-images.githubusercontent.com/94300963/232187195-40787104-92f2-4f94-bcde-bbe56b7dac73.mov